src/styles/1-helpers/_mixins.scss

```diff
@@ -0,0 +1,5 @@
+@mixin flex-centering($just-cont) {
```

Comment:
Your `flex-centering` mixin doesn't do centering, so it would be better to call it `flex-align` (i.e. a more generic name).